### DISCUSSION TITLE
How does this work?

### DISCUSSION THREAD

**Discussion by SapoKR** [miguelgrinberg/python-engineio#314] [Q&A]
I want to send to 42x["route", {"data":"data"}] in AsyncClient, but it sends to 4["route", {"data":"data"}] when I send. What should I change?
I tried adding 2x manually, but I get it as a string, not as a list.

**Comment by miguelgrinberg** [comment]
This package is a high level implementation of the Socket.IO protocol. Low level access to the packet encoding is not provided. If you want to encode your own packets, I suggest you use a WebSocket client, and then you will be able to encode and decode packets on your own and make them be what you need exactly.
